sphinx_gallery/_static/sg_gallery.css

```diff
@@ -74,23 +74,26 @@ thumbnail with its default link Background color */
 
 .sphx-glr-script-out {
   color: #888;
-  margin: 0;
+  display: flex;
+  gap: 0.5em;
 }
-p.sphx-glr-script-out {
-    padding-top: 0.7em;
+.sphx-glr-script-out::before {
+  content: "Out:";
+  line-height: 1.4;
+  padding-top: 10px;
 }
 .sphx-glr-script-out .highlight {
   background-color: transparent;
-  margin-left: 2.5em;
-  margin-top: -2.9em;
+  flex-grow: 1;
+  overflow: auto;
```

Comment:
This keeps the `div` from overflowing its flex container.

Reply:
Nitpick - could you add this as a comment in the css for people not familiar with css (like me)?